devimint/src/util.rs

```diff
@@ -1218,19 +1220,24 @@ pub async fn use_matching_fedimint_cli_for_dkg() -> Result<(String, String)> {
         fedimint_cli = crate::util::get_fedimint_cli_path().join(" "),
         datadir = crate::vars::utf8(&fm_client_dir_path_buf)
     );
-    std::env::set_var(FM_MINT_CLIENT_ENV, fm_mint_client);
+    // TODO: Audit that the environment access only happens in single-threaded code.
+    unsafe { std::env::set_var(FM_MINT_CLIENT_ENV, fm_mint_client) };
 
     Ok((original_fedimint_cli_path, original_fm_mint_client))
 }
 
 /// Sets the fedimint-cli and mint client alias
 pub fn use_fedimint_cli(original_fedimint_cli_path: String, original_fm_mint_client: String) {
-    std::env::set_var(
-        FM_FEDIMINT_CLI_BASE_EXECUTABLE_ENV,
-        original_fedimint_cli_path,
-    );
+    // TODO: Audit that the environment access only happens in single-threaded code.
```

Comment:
@bradleystachurski I remember pointing out recently these will be be `unsafe` soon. Not a big deal, especially in devimint.

Reply:
Excellent improvement for 2024.

Do you think it's worth considering a different approach in devimint? I'll create a tracking issue if so.

Reply:
@bradleystachurski In principle it should always be possible to carry stuff like that in some variables, and set only when starting child commands.